dev-scheme/guile/guile-3.0.10-r101.ebuild

```diff
@@ -49,6 +55,7 @@ GUILE_INFODIR="${GUILE_DATA}"/info
 src_configure() {
 	# see bug #676468
 	mv prebuilt/32-bit-big-endian{,.broken} || die
+	eautoreconf
```

Comment:
This could do with a comment noting it's only for the two patches in 3.0.10 for 32-bit support.

Reply:
But actually, is it needed at all? The patch is needed because of the `GUILE_OPTIMIZATIONS` change, but we delete the binary above.

But maybe we _do_ want the patch + eautoreconf so we can delete that deletion and speed up build on 32-bit BE.

(Whether or not we should drop the prebuilt binaries entirely is another matter but I'm talking about dropping the special-casing for now.)

Reply:
Yes, it was necessary.  Without this `stage0/Makefile.am` patch, I still got a segfault, even with the other patch present.

Reply:
... because "little-endian" != "big-endian", doh.

(That said, there's still a route to dropping the `rm`, but let's handle that another time).

Nevermind, thanks!